providers/cxgb4/dev.c

```diff
@@ -196,6 +180,23 @@ static struct verbs_context *c4iw_alloc_context(struct ibv_device *ibdev,
 				context->status_page->write_cmpl_supported;
 	}
 
+	rhp->chip_version = CHELSIO_CHIP_VERSION(attr.vendor_part_id >> 8);
+	switch (rhp->chip_version) {
+	case CHELSIO_T6:
+		PDBG("%s T6/T5/T4 device\n", __FUNCTION__);
```

Comment:
Macro PDBG worries me, it is a wrapper to syslog(LOG_DEBUG, ....), which is not visible without enabling full debug mode for systemd.

I afraid that the right thing will be to delete it.

Reply:
Yes, I have been thinking of removing it as part of my clean-up changes.
I am thinking of changing it to env variable based prints instead.
Do you have any better suggestions?
Thanks @rleon

Reply:
No prints are supposed to be in library, I think that combination of general environment variable with prints to syslog is the right thing to do.

Se this commit https://github.com/linux-rdma/rdma-core/pull/554/commits/ef8aca40b140d234aadbdaedbb6a2199dd2dce6e

mlx5_dbg should be removed too.

Reply:
LOG_DEBUG works fine, it depends on the journald configuration in /etc/ if it gets recorded or not.

Reply:
How will you enable log_debug for a specific application?